modules/local/merge_predictions.nf

```diff
@@ -0,0 +1,54 @@
+process MERGE_PREDICTIONS {
+    label 'process_single'
+    tag "${metadata.sample}"
+
+    conda "bioconda::mhcgnomes=1.8.4"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mhcgnomes:1.8.4--pyh7cba7a3_0' :
+        'quay.io/biocontainers/mhcgnomes:1.8.4--pyh7cba7a3_0' }"
+
+    input:
+    tuple val(metadata), path(prediction_files), path(metadata_file)
+
+    output:
+    tuple val(metadata), path("*.tsv"), emit: merged
+    path "versions.yml", emit: versions
+
+    script:
+    def output = prediction_files.first().baseName.split("_").dropRight(2).join("_")
+    def min_length = (metadata.mhc_class == "I") ? params.min_peptide_length_mhc_I : params.min_peptide_length_mhc_II
+    def max_length = (metadata.mhc_class == "I") ? params.max_peptide_length_mhc_I : params.max_peptide_length_mhc_II
+
+    def syfpeithi_threshold = params.syfpeithi_threshold ? "--syfpeithi_threshold ${params.syfpeithi_threshold}" : ""
```

Comment:
I think we do filter on thresholds using a json file with `--tool_thresholds`  https://github.com/nf-core/epitopeprediction/blob/b0f0d7a5b3cd352e548901e4cc19937c73add465/bin/epaa.py#L1388

and if we want to filter based on affinity we specify `--use_affinity_thresholds`. Maybe parse these two parameters in the not not yet existing python file instead of defining this 4 thresholds